apps/web/src/components/QualifiedRecruitmentCard/QualifiedRecruitmentCard.tsx

```diff
@@ -0,0 +1,343 @@
+  Heading,
+  HeadingRank,
+  Pill,
+  Separator,
+  incrementHeadingRank,
+} from "@gc-digital-talent/ui";
+import { notEmpty } from "@gc-digital-talent/helpers";
+import { getLocalizedName, getSkillCategory } from "@gc-digital-talent/i18n";
+
+import { PoolCandidate, SkillCategory } from "~/api/generated";
+import { categorizeSkill } from "~/utils/skillUtils";
+
+import { getQualifiedRecruitmentInfo, joinDepartments } from "./utils";
+import RecruitmentAvailabilityDialog from "../RecruitmentAvailabilityDialog/RecruitmentAvailabilityDialog";
+
+interface QualifiedRecruitmentCardProps {
+  candidate: PoolCandidate;
+  headingLevel?: HeadingRank;
+}
+
+const QualifiedRecruitmentCard = ({
+  candidate,
+  headingLevel = "h2",
+}: QualifiedRecruitmentCardProps) => {
+  const intl = useIntl();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [linkCopied, setLinkCopied] = React.useState<boolean>(false);
+  const contentHeadingLevel = incrementHeadingRank(headingLevel);
+  const {
+    title,
+    isQualified,
+    isOngoing,
+    statusPill,
+    availability: { icon: AvailabilityIcon, ...availability },
+  } = getQualifiedRecruitmentInfo(candidate, intl);
+
+  const departments = joinDepartments(
+    candidate.pool?.team?.departments ?? [],
+    intl,
+  );
+
+  // NOTE: Until we store assessed skills, we will just be displayed all essential skills
+  const categorizedSkills = categorizeSkill(
+    candidate.pool.essentialSkills?.filter(notEmpty) ?? [],
+  );
+
+  /** Reset link copied after 3 seconds */
+  React.useEffect(() => {
+    if (linkCopied) {
+      setTimeout(() => {
+        setLinkCopied(false);
+      }, 3000);
+    }
+  }, [linkCopied, setLinkCopied]);
+
+  return (
+    <div
+      data-h2-border-left="base(x.5 solid secondary)"
+      data-h2-padding="base(x1 x1 x.5 x1)"
+      data-h2-shadow="base(larger)"
+      data-h2-radius="base(0 rounded rounded 0)"
+      data-h2-background-color="base(white) base:dark(black)"
+    >
+      <div
+        data-h2-display="base(flex)"
+        data-h2-align-items="base(center)"
+        data-h2-justify-content="base(space-between)"
+        data-h2-gap="base(0 x1)"
+      >
+        <Heading level={headingLevel} size="h6" data-h2-margin="base(0)">
+          {title.html}
+        </Heading>
+        <Pill mode="outline" color={statusPill.color}>
+          {isQualified ? (
+            <span
+              data-h2-display="base(flex)"
+              data-h2-align-items="base(center)"
+              data-h2-gap="base(0 x.25)"
+            >
+              <ShieldCheckIcon
+                data-h2-width="base(1rem)"
+                data-h2-height="base(auto)"
+              />
+              <span>{statusPill.text}</span>
+            </span>
+          ) : (
+            statusPill.text
+          )}
+        </Pill>
+      </div>
+      <p
+        data-h2-color="base(secondary.darker)"
+        data-h2-margin="base(x.5 0 x1 0)"
+      >
+        {isOngoing
```

Comment:
oh am doing the samething .. I think we are overlapping ..will base mine of yours then.
Do you mind taking this out into a utility function something like getRecruitmentType?

Reply:
Yeah, of course! I can do that and let you know when it is completed 👍